app/views/ClaimByAlternativeView.scala.html

```diff
@@ -31,6 +34,12 @@
 
         @components.heading("claimByAlternative.heading")
 
+        <p>@messages("claimByAlternative.paragraph")
+          <a href="@frontendAppConfig.p87ClaimOnlineUrl">@messages("claimByAlternative.onlineFormLink")</a>
+          @messages("site.or")
+          <a href="@frontendAppConfig.p87ClaimByPostUrl">@messages("claimByAlternative.claimByPostLink")</a>.
+        </p>
```

Comment:
Can this be done properly with placeholders as in https://github.com/hmrc/employee-expenses-frontend/blob/master/app/views/authenticated/NoCodeChangeView.scala.html